tests/test_wires.py

```diff
@@ -86,28 +110,31 @@ def test_equality(self):
         assert wires1 != wires2
         assert wires1 == wires3
 
-    def test_is_ordered(self):
-        """Tests that a Wires object is not equal to another Wires object with a different ordering of the indices."""
+    @pytest.mark.parametrize("iterable", [[4, 1, 0, 3],
+                                          ['a', 'b', 'c']])
+    def test_length(self, iterable):
+        """Tests that a Wires object returns the correct length."""
 
-        wires1 = Wires([1, 2, 3])
-        wires2 = Wires([3, 2, 1])
-        assert wires1 != wires2
+        wires = Wires(iterable)
+        assert len(wires) == len(iterable)
 
-    def test_length(self):
-        """Tests that a Wires object returns the correct length."""
+    def test_contains(self, ):
+        """Tests the __contains__() method."""
 
-        wires = Wires([1, 2, 3, 4, 5])
-        assert len(wires) == 5
+        wires = Wires([0, 1, 2, 3])
+        assert Wires([0, 3]) in wires
+        assert Wires([1]) in wires
+        assert not Wires([0, 4]) in wires
+        assert not Wires([4]) in wires
```

Comment:
Can we also add an assert where the LHS is not itself a wires object? e.g., `0 in wires`?

Reply:
and also a list object!

Reply:
Oh yes!